python/doc/theory/meta_modeling/chaos_basis.rst

```diff
@@ -3,132 +3,191 @@
+where :math:`\idx = (\alpha_1,\dots,\alpha_{n_x}) \in \Nset^{n_X}` is
+a multi-index representing the marginal polynomial degrees.
 
-**Building of the PC basis – dependent random variables**
+PC basis with dependent random variables
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-| In case of *dependent* variables, it is possible to build up an
-  orthonormal basis as follows:
+In case of *dependent* variables, it is possible to build up an
+orthonormal basis as follows (see [soizeghanem2004]_):
 
-  .. math::
+.. math::
     :label: 3.010seven
 
-      \psi_{\idx}(\vect{x}) \, \, = \,\,  K(\underline{x}) \;\prod_{i=1}^M \pi^{(i)}_{\alpha_{i}}(x_{i})
+    \psi_{\idx}(\vect{z})
+    = \left( \frac{f_1(z_1) \cdots f_{n_X}(z_{n_X})}{f_{\vect{Z}}(\vect{z})}\right)^{\frac{1}{2}}
+    \prod_{i=1}^{n_X} \pi^{(i)}_{\alpha_{i}}(z_i).
 
-  where :math:`K(\underline{x})` is a function of the copula of
-  :math:`\vect{X}`. Note that such a basis is no longer polynomial. When
-  dealing with independent random variables, one gets
-  :math:`K(\underline{x})=1` and each basis element may be recast as in
-  :eq:`3.010six`. Determining :math:`K(\underline{x})` is usually
-  computationally expensive though, hence an alternative strategy for
-  specific types of input random vectors.
+Notice that such a basis is no longer made of polynomials. When
+dealing with independent random variables, each basis element may be recast as in
+:eq:`3.010six`.
+Computing :math:`\psi_{\idx}(\vect{z})` can be expensive.
+This is the reason why we present below other methods for specific types
+of input random vectors.
 
-| If :math:`\vect{X}` has an elliptical copula instead of an independent
-  one, it may be recast as a random vector :math:`\vect{U}` with
-  independent components using a suitable mapping
-  :math:`T : \vect{X} \mapsto \vect{U}` such as the Nataf transformation.
-  The so-called Rosenblatt transformation may also be applied in case
-  of a Gaussian copula . Thus the model response :math:`Y` may be
-  regarded as a function of :math:`\vect{U}` and expanded onto a
-  *polynomial* chaos expansion with basis elements cast as in :eq:`3.010six`.
+If :math:`\vect{X}` has an elliptical copula instead of an independent
+one, it may be recast as a random vector :math:`\vect{Z}` with
+independent components using a suitable mapping
+:math:`T : \vect{X} \mapsto \vect{Z}` such as the Nataf transformation
+for example.
+The so-called Rosenblatt transformation may also be applied in case
+of a Gaussian copula.
 
-| **Link with classical deterministic polynomial approximation**
+Link with classical deterministic polynomial approximation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 In a deterministic setting (i.e. when the input parameters are
 considered to be deterministic), it is of common practice to substitute
 the model function :math:`h` by a polynomial approximation over its
 whole domain of definition as shown in . Actually this approach is
```

Comment:
as shown where?

Reply:
I do not know. I remove this part of the sentence.